code/model/DNProject.php

```diff
@@ -286,6 +287,20 @@ public function canDownloadArchive($member = null) {
 	}
 
 	/**
+	 * This is a permission check for the front-end only.
```

Comment:
This seems to be not untrue? DNRoot::doCreateEnvironment uses that check for backend processing.